modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -125,27 +130,28 @@ object FormInputEV {
 
     def render(p: Props[EV, A], s: State): VdomNode = {
 
-      val validationError: js.UndefOr[Label] =
-        s.errors
-          .map(e =>
-            Label(content = e.toList.mkString(","),
-                  clazz = p.errorClazz,
-                  pointing = p.errorPointing
-            )(^.position.absolute)
-          )
-          .orUndefined
+      def errorLabel(errors: NonEmptyChain[NonEmptyString]): js.UndefOr[ShorthandB[Label]] = {
+        val vdoms = errors.toList.map[VdomNode](_.value)
+        val list  = vdoms.head +: vdoms.tail.flatMap[VdomNode](e => List(<.br, <.br, e))
+        Label(
```

Comment:
Can you use `mkTagMod` here?

Reply:
That's the first thing I tried, but it returns a TagMod and Label content needs a VdomNode. Maybe there is a way of going from TagMod => VdomNode?

Reply:
Hmmm. Not that I'm aware of.